github-config.yaml

```diff
@@ -419,3 +427,13 @@ orgs:
         members:
           - HumairAK
           - durandom
+      sre:

```

Comment:
could we come up with a more descriptive name?

or just `sig-sre`? this would imply (via sig) that it's a special interest group, and not the team doing the SRE work in the community.

Reply:
`sig-sre` would imply that within op1st a new SIG has constituted with the name of "SIG SRE" (like SIG ops and SIG community) and this is that group. But it's not and I suspect SIG SRE might constitute at some point, which would leave a name conflict.

I picked `sre` as a name that in this context is supposed to mean "group that creates/maintains SRE–related materials" and not "group that does SRE work for the op1st cloud", since SIG Ops is the one that's tasked with handling ops things in the cloud (see [charter](https://github.com/operate-first/community/blob/main/sig-operations/charter.md)).

I really can't really come up with a better name at this point.

Reply:
@durandom if you're convinced feel free to `/ approve` for merge!

Reply:
ok, I dont have a better name either 🤷